client/rechallenge/transformers.js

```diff
@@ -18,7 +19,13 @@ import {
 } from '../../common/utils/polyvinyl.js';
 import castToObservable from '../../common/app/utils/cast-to-observable.js';
 
-const babelOptions = { presets: [ presetEs2015, presetReact ] };
+/* NOTE: About stage-2 preset:
+ * Add stage-2 transform to enable es2016 class properties transform,
+ * i.e. to allow React class methods to be written as arrow functions.
+ * This is only needed in the React challenges, and since these are
+ * isolated in the buildChallengeEpic they could be fed with separate
+ * Babel presets. However, enabling everywhere is faster for now: */
+const babelOptions = { presets: [ presetEs2015, presetReact, stage2 ] };
```

Comment:
I'd prefer we not teach anything that has not made it into the spec. class properties is not yet part of the spec and not specifically related to react.